src/DataDepsGenerators.jl

```diff
@@ -11,6 +11,7 @@ struct Metadata
     website::String
     description::String
     dataurls::Vector
+    datachecksums::Vector
```

Comment:
make this unrestricted in type.
using `dataurls::Any`
because there are many options for a checksum.

 - `nothing`,   
 - `(md5, "234213542ababab23...")`,  
 - `"234213542ababab23..."`
 - Vectors of any combination of valid forms